Wordle.Api/Wordle.Api/Controllers/LeaderboardController.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class LeaderboardController : ControllerBase
+{
+    private readonly LeaderboardService _leaderboardService;
+
+    public LeaderboardController(LeaderboardService leaderboardService)
+    {
+        _leaderboardService = leaderboardService;
+    }
+
+    [HttpGet("GetTopTenScores")]
```

Comment:
Typically we don't put the action in the endpoint name... This would be better named as 
```suggestion
    [HttpGet("TopTenScores")]
```

or as 
```suggestion
    [HttpGet("TopScores")]
```
if you took a number of scores to retrieve as a parameter